### PR TITLE
Add conservative stack scanning bug found in Chromium

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2995,3 +2995,11 @@ @misc{richards99bench
   url={https://www.cl.cam.ac.uk/~mr10/Bench.html},
   urldate={2023-03-01}
 }
+
+@misc{chromium20cssbug,
+    title = {Octane benchmark suite},
```
Comment (jacob-hughes) [inline softdev.bib:3000]: Ugh, copy paste bug. I'll push a fix

Comment (jacob-hughes) [reply softdev.bib:3000]: Actually, I'm a little unsure what to call this. Can I use the lengthy title of the bug report? 

"base::Optional<T> when compiled with clang leads to memory layout that is incompatible with conservative stack scanning"

Comment (ltratt) [reply softdev.bib:3000]: Good question. Let's try that as the title and see how we feel in a day or so!

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed

**Comment by jacob-hughes** [comment]
84540584ca4bc302fdea69f6ad2762f500c8e7a6

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed

**Pull request merged**